contracts/gateway.yaml

```diff
@@ -3981,4 +4040,20 @@ components:
         error: 
           $ref: '#/components/schemas/Error'
         resp:
-          $ref: '#/components/schemas/RequestReference'
\ No newline at end of file
+          $ref: '#/components/schemas/RequestReference'
+    ServiceProfileResponse:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        type:
+          $ref: '#/components/schemas/ServiceType'
+        active:
+          type: boolean
+      xml:
+        name: ServiceProfileResponse
+      ServiceType:
```

Comment:
I think the indentation of the "ServiceType" is wrong. You can validate using editor.swagger.io

Reply:
Done